backend/dataall/modules/dataset_sharing/services/share_permissions.py

```diff
@@ -46,13 +46,13 @@
 ENVIRONMENT_INVITED.append(CREATE_SHARE_OBJECT)
 ENVIRONMENT_INVITED.append(LIST_ENVIRONMENT_SHARED_WITH_OBJECTS)
 ENVIRONMENT_INVITATION_REQUEST.append(CREATE_SHARE_OBJECT)
-ENVIRONMENT_INVITATION_REQUEST.append(LIST_ENVIRONMENT_SHARED_WITH_OBJECTS)
+ENVIRONMENT_INVITED_DEFAULT.append(LIST_ENVIRONMENT_SHARED_WITH_OBJECTS)
 ENVIRONMENT_ALL.append(CREATE_SHARE_OBJECT)
 ENVIRONMENT_ALL.append(LIST_ENVIRONMENT_SHARED_WITH_OBJECTS)
 
 RESOURCES_ALL.extend(SHARE_OBJECT_ALL)
 for perm in SHARE_OBJECT_ALL:
     RESOURCES_ALL_WITH_DESC[perm] = perm
 
-RESOURCES_ALL_WITH_DESC[CREATE_SHARE_OBJECT] = 'Request datasets access for this environment'
-RESOURCES_ALL_WITH_DESC[LIST_ENVIRONMENT_SHARED_WITH_OBJECTS] = "List datasets shared with this environments"
+RESOURCES_ALL_WITH_DESC[CREATE_SHARE_OBJECT] = 'Create dataset Share requests for this environment'
+RESOURCES_ALL_WITH_DESC[LIST_ENVIRONMENT_SHARED_WITH_OBJECTS] = 'LIST_ENVIRONMENT_SHARED_WITH_OBJECTS'
```

Comment:
The description used to be nice :(

Reply:
Yes, but this way we can clearly differentiate those that are plotted in the UI from those that are used internally. I just wanted to be consistent